components/dashboard/src/projects/Prebuild.tsx

```diff
@@ -78,13 +81,26 @@ export default function () {
         </div>)
     };
 
+    const onInstanceUpdate = (instance: WorkspaceInstance) => {
+        setPrebuildInstance(instance);
+    }
+
     useEffect(() => { document.title = 'Prebuild — Gitpod' }, []);
 
     return <>
         <Header title={renderTitle()} subtitle={renderSubtitle()} />
         <div className="lg:px-28 px-10 mt-8">
-            <div className="h-96 rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col">
-                <PrebuildLogs workspaceId={prebuild?.info?.buildWorkspaceId}/>
+            <div className="rounded-xl overflow-hidden bg-gray-100 dark:bg-gray-800 flex flex-col">
```

Comment:
**issue**: Could not get any prebuild log output here. Expected? 😭

Reply:
It took some time the first time I opened it. But I, at last, got logs. I think this is unrelated to this PR.

Reply:
Yes, sometimes logs don't show up at all ("Error: could not find headless logs for 1234"). Sometimes, they don't show up live, but eventually the entire logs show up after the prebuild is complete.

This morning I also had live logs. It's kind of a shame that this isn't reliable.

Reply:
We need to look into it (in a separate PR). Can we merge this? 😁